tests/unittests/text-translator-quantization.test

```diff
@@ -0,0 +1,5 @@
+RUN: %text-translator -m %build_dir/en2gr  -keep-original-precision-for-nodes=Add -dump-profile=%t.yaml -quantization-schema=symmetric <<< "My name is Bob ." && \
+RUN: (%text-translator -m %build_dir/en2gr  -keep-original-precision-for-nodes=Add -load-profile=%t.yaml <<< "My name is Bob ." | FileCheck %s)
```

Comment:
`%s` means the source file, i.e., this file.
So `... | FileCheck %s` means run `FileCheck` with the output of `...` and with this file as input. `FileCheck` is the tool that looks for `CHECK` directives and do the pattern matching.